src/lightly_train/_modules/teachers/dinov2/layers/block.py

```diff
@@ -0,0 +1,294 @@
+#
+# Copyright (c) Lightly AG and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the license found in the
+# LICENSE file in the root directory of this source tree.
+#
+
+# References:
+#   https://github.com/facebookresearch/dino/blob/master/vision_transformer.py
+#   https://github.com/rwightman/pytorch-image-models/tree/master/timm/layers/patch_embed.py
+
+import logging
+import os
+from typing import Any, Callable, Dict, List, Tuple
```

Comment:
```suggestion
from typing import Any, Callable
```

You can use `dict, list, tuple` when importing `from __future__ import annotations`

Reply:
Ah no then leave it as is 👍🏼